README.md

```diff
@@ -83,52 +85,46 @@ Where `source` is the file location.
 Having the following workflow instance:
 
 ```typescript
-    const workflow = new WorkflowBuilder()
-        .withId("helloworld")
-        .withVersion("1.0")
-        .withName("Hello World Workflow")
-        .withDescription("Inject Hello World")
-        .withStart("Hello State")
-        .withStates([new InjectStateBuilder()
-            .withName("Hello State")
-            .withData({
-                "result": "Hello World!"
-            })
-            .withEnd(true).build()])
-        .build();
+const workflow = workflowBuilder()
+  .id("helloworld")
+  .version("1.0")
+  .name("Hello World Workflow")
+  .description("Inject Hello World")
+  .start("Hello State")
+  .states([injectstateBuilder()
```

Comment:
Sorry if this is a stupid question, but why do you call these things "Builder"? 

To me something like this would read a little easier: 

```json
const workflow = Workflow()
  .id("helloworld")
  .version("1.0")
  .name("Hello World Workflow")
  .description("Inject Hello World")
  .start("Hello State")
  .states([InjectState()
    .name("Hello State")
    .data({
      "result": "Hello World!"
    })
    .end(true)
    .build()
  ])
  .build();
```

Reply:
There is no stupid question, always feel free to ask.

These are called builder because they are factory functions returning an object of type `Builder`. A builder object is a proxy that prevents setters and that intercepts getters to return function that will internally set a value to the underlying object and return the builder (so it's fluent). So `Workflow` or `InjectState` are types or interfaces (depending of the specification requirements) and `workflowBuider` or `injectionBuilder` are factories that return a builder/proxy.